File: tests/testdata/codegen/transpiled_examples/getting-started-pp/python/getting-started.py
import pulumi
import pulumi_aws as aws

mybucket = aws.s3.Bucket("mybucket", website={
    "index_document": "index.html",
})
indexhtml = aws.s3.BucketObject("indexhtml",
    bucket=mybucket.id,
    source=pulumi.StringAsset("<h1>Hello, world!</h1>"),
    acl="public-read",
    content_type="text/html")
pulumi.export("bucketEndpoint", mybucket.website_endpoint.apply(lambda website_endpoint: f"http://{website_endpoint}"))

File: tests/testdata/codegen/transpiled_examples/getting-started-pp/dotnet/getting-started.cs
using System.Collections.Generic;
using System.Linq;
using Pulumi;
using Aws = Pulumi.Aws;

return await Deployment.RunAsync(() => 
{
    var mybucket = new Aws.S3.Bucket("mybucket", new()
    {
        Website = new Aws.S3.Inputs.BucketWebsiteArgs
        {
            IndexDocument = "index.html",
        },
    });

    var indexhtml = new Aws.S3.BucketObject("indexhtml", new()
    {
        Bucket = mybucket.Id,
        Source = new StringAsset("<h1>Hello, world!</h1>"),
        Acl = "public-read",
        ContentType = "text/html",
    });

    return new Dictionary<string, object?>
    {
        ["bucketEndpoint"] = mybucket.WebsiteEndpoint.Apply(websiteEndpoint => $"http://{websiteEndpoint}"),
    };
});

